CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**Want to contribute? Great!** 
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples... 
+But first, read this page (including the small print at the end).
+
+## Legal
+
+All original contributions to Jandex licensed under the
+[ASL - Apache License](https://www.apache.org/licenses/LICENSE-2.0),
+version 2.0 or later, or, if another license is specified as governing the file or directory being
+modified, such other license.
```

Comment:
Any reason why you dropped the DCO paragraph here? I don't really understand the difference, just asking :-)

Reply:
There is no dco.txt file in Jandex so I was not sure if it applied.
If you want I can add https://github.com/smallrye/smallrye-mutiny/blob/main/dco.txt to this PR and put back the DCO paragraph

Reply:
Please do, thanks!

Reply:
done